src/node_dotenv.cc

```diff
@@ -55,11 +57,17 @@ void Dotenv::SetEnvironment(node::Environment* env) {
       env->env_vars()->Set(
           isolate,
           v8::String::NewFromUtf8(
-              isolate, key.data(), NewStringType::kNormal, key.size())
-              .ToLocalChecked(),
```

Comment:
These changes seems to be unnecessary.

Reply:
This is my formatter wrapping the lines in a pretty format for under 80 chars. 
![image](https://github.com/nodejs/node/assets/136755106/efcda60e-a847-4030-8dcd-4e1e3e808083)

Reply:
if you want it put back or revert I can but that's the reason

Reply:
We are using clang-format. You can run linter with make lint-cpp and make format-cpp. 

These kinds of changes bloat the git history and makes it harder to use git blame.